src/main/kotlin/com/stake/mercariapplication/controller/LoginController.kt

```
@@ -0,0 +1,37 @@
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.validation.BindingResult
+import org.springframework.validation.annotation.Validated
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RestController
+import org.springframework.web.server.ResponseStatusException
+
+@RestController
+@Authorize
+class LoginController() {
```

detekt

[warning] An empty default constructor can be removed.

(detekt.empty-blocks.EmptyDefaultConstructor)